tests/function/gtBuffer.cpp

```diff
@@ -215,6 +215,11 @@ TEST(LibopaecBufCommonALL, Prep0B) {
 
   EXPECT_EQ(FPGA_INVALID_PARAM, fpgaReleaseBuffer(h, wsid));
 #endif
+  UNUSED_PARAM(buf_len);
+  UNUSED_PARAM(buf_addr);
+  UNUSED_PARAM(wsid);
+  UNUSED_PARAM(flags);
+
```

Comment:
Can this be rewritten to something like:
```C++
#ifdef BUILD_ASE
  UNUSED_PARAM(buf_len);
  //...



#else
  buf_len =  0;
  flags = 0;
  // ...
#endif
```

Reply:
Yes, I think that can be done.. let me make those changes and push a new commit.